wasmbinding/bindings/msg.go

```diff
@@ -261,13 +261,18 @@ type Dex struct {
 // MsgPlaceLimitOrder is a copy dextypes.MsgPlaceLimitOrder with altered ExpirationTime field,
 // it's a preferable way to pass timestamp as unixtime to contracts
 type MsgPlaceLimitOrder struct {
-	Creator          string   `json:"creator,omitempty"`
-	Receiver         string   `json:"receiver,omitempty"`
-	TokenIn          string   `json:"token_in,omitempty"`
-	TokenOut         string   `json:"token_out,omitempty"`
-	TickIndexInToOut int64    `json:"tick_index_in_to_out,omitempty"`
-	AmountIn         math.Int `json:"amount_in"`
-	OrderType        string   `json:"order_type,omitempty"`
+	Creator  string `json:"creator,omitempty"`
+	Receiver string `json:"receiver,omitempty"`
+	TokenIn  string `json:"token_in,omitempty"`
+	TokenOut string `json:"token_out,omitempty"`
+	// Deprecated: tick_index_in_to_out will be removed in future release; limit_sell_price should be used instead.
+	TickIndexInToOut int64  `json:"tick_index_in_to_out,omitempty"`
+	LimitSellPrice   string `json:"limit_sell_price,omitempty"`
+	// LimitSellPriceExp can be used to shift LimitSellPrice to a smaller decimal by an order of 10^LimitSellPriceExp
+	// ie. LimitSellPrice: 1.123 && LimitSellPriceExp: 5 => 1.123 / (10^5) = 0.00001123
+	LimitSellPriceExp uint64   `json:"limit_sell_price_exp,omitempty"`
```

Comment:
What the aim of the field, why cant you set desired price with a single `LimitSellPrice` instead of `LimitSellPrice`+`LimitSellPriceExp`. Why the field introduced only for wasm bindings?